src/osd/ECBackend.cc

```diff
@@ -1231,7 +1230,7 @@ void ECBackend::handle_sub_read_reply(
 	    err = rop.complete[iter->first].errors.begin()->second;
             rop.complete[iter->first].r = err;
 	  } else {
-	    get_parent()->clog_error() << __func__ << ": Error(s) ignored for "
+	    get_parent()->clog_error() << "Error(s) ignored for "
```

Comment:
Since we're continuing without needing admin intervention, should be warn level - it'll be caught during the next deep scrub anyway